src/test/java/uk/nhs/prm/deduction/e2e/tests/RepositoryE2ETests.java

```diff
@@ -196,51 +188,103 @@ void shouldVerifyThatASmallEhrXMLIsUnchanged() {
-                    .withNodeFilter(TestUtils::excludeComparisons)
-                    .withDifferenceEvaluator(DifferenceEvaluators.chain(DifferenceEvaluators.Default,
-                            DifferenceEvaluators.downgradeDifferencesToEqual(ComparisonType.XML_STANDALONE)))
-                    .checkForSimilar().build();
+        String gp2gpMessengerPayload = getPayloadOptional(gp2gpMessage.body()).orElseThrow();
+        String smallEhrPayload = getPayloadOptional(smallEhr).orElseThrow();
+        LOGGER.info("Payload from gp2gpMessenger: {}", gp2gpMessengerPayload);
+        LOGGER.info("Payload from smallEhr: {}", smallEhrPayload);
 
-            assertFalse(myDiff.toString(), myDiff.hasDifferences());
-
-        } catch (JSONException exception) {
-            LOGGER.error(exception);
-            throw new Error(exception);
-        }
-    }
-
-    @Test
-    public void given2XMLsWithDifferences_whenTestsSimilarWithDifferenceEvaluator_thenCorrect() {
-        // helper test case for building the `excludeComparisons` function
-//        final String control = "<agentOrganizationSDS classCode=\"ORG\" determinerCode=\"INSTANCE\"><id root=\"1.2.826.0.1285.0.1.10\" extension=\"B85002\"/></agentOrganizationSDS>";
-//        final String test =    "<agentOrganizationSDS classCode=\"ORG\" determinerCode=\"INSTANCE\"><id root=\"1.2.826.0.1285.0.1.10\" extension=\"M85019\"/></agentOrganizationSDS>";
-//        final String control = "<device classCode=\"DEV\" determinerCode=\"INSTANCE\"><id root=\"1.2.826.0.1285.0.2.0.107\" extension=\"200000001613\"/></device>";
-//        final String test =    "<device classCode=\"DEV\" determinerCode=\"INSTANCE\"><id root=\"1.2.826.0.1285.0.2.0.107\" extension=\"200000000149\"/></device>";
-//        final String control = "<id root=\"DF91D420-DDC7-11ED-808B-AC162D1F16F0\"/>";
-//        final String test    = "<id root=\"DFBA6AC0-DDC7-11ED-808B-AC162D1F16F0\"/>";
-        final String control = "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\" ?><RCMR_IN030000UK06 xmlns=\"urn:hl7-org:v3\" xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xsi:schemaLocation=\"urn:hl7-org:v3 ..\\Schemas\\RCMR_IN030000UK06.xsd\"></RCMR_IN030000UK06>";
-        final String test    = "<?xml version=\"1.0\" encoding=\"UTF-8\" ?><RCMR_IN030000UK06 xmlns=\"urn:hl7-org:v3\" xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xsi:schemaLocation=\"urn:hl7-org:v3 ..\\Schemas\\RCMR_IN030000UK06.xsd\"></RCMR_IN030000UK06>";
-
-
-        Diff myDiff = DiffBuilder.compare(control).withTest(test)
-                .withNodeFilter(TestUtils::excludeComparisons)
-                .withDifferenceEvaluator(DifferenceEvaluators.chain(DifferenceEvaluators.Default,
-                        DifferenceEvaluators.downgradeDifferencesToEqual(ComparisonType.XML_STANDALONE)))
-                .checkForSimilar().build();
+        Diff myDiff = TestUtils.comparePayloads(gp2gpMessengerPayload, smallEhrPayload);
```

Comment:
I think we should import the `comparePayloads` function as a static import as it is referred to many times within this file.

Example:
`import static uk.nhs.prm.deduction.e2e.utility.TestUtils.comparePayloads`

This way you don't need to use `TestUtils.comparePayloads();`, you can just use `comparePayloads();`